security/cert_manager_operator/cert-manager-operator-install.adoc

```diff
@@ -6,6 +6,11 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
+[IMPORTANT]
+====
+The {cert-manager-operator} version 1.15 or later supports the `AllNamespaces`, `SingleNamespace` and `OwnNamespace` installation mode. Earlier versions, such as 1.14, support only the `SingleNamespace` and `OwnNamespace` installation modes.
```

Comment:
Needs an Oxford comma per the [ISG](https://www.ibm.com/docs/en/ibm-style?topic=punctuation-commas#commas-between-items-in-a-series):
```suggestion
The {cert-manager-operator} version 1.15 or later supports the `AllNamespaces`, `SingleNamespace`, and `OwnNamespace` installation mode. Earlier versions, such as 1.14, support only the `SingleNamespace` and `OwnNamespace` installation modes.
```